ADA/databricks_rstudio.qmd

```diff
@@ -0,0 +1,70 @@
+---
+  title: "Setup Databricks with RStudio"
+---
+
+<p class="text-muted">
+
+The following instructions set up an ODBC connection between your laptop and your DataBricks cluster, which can then be used in R/RStudio to query data using an ODBC based package.
+
+</p>
```

Comment:
Think the muted attribute is failing with this being on separate lines, so the text isn't faded like other subtitles, might work if you smush it all to line 5 for the subheading?